config/features.rb

```diff
@@ -7,46 +7,6 @@
     default: false,
     description: 'Debug mode'
 
-  feature :local_browse,
-    default: false,
-    description: 'Enables local browsing of paginated results'
-
-  feature :check_online,
-    default: false,
-    description: 'Enables button for EDS supplied non-subscribed SFX links'
-
-  feature :local_full_record,
-    default: ENV['LOCAL_FULL_RECORD'],
-    description: 'Enables local full record instead of sending to EDS UI'
-
-  feature :pride,
-    default: ENV['PRIDE'],
-    description: 'Enables rainbows for records with LGBT subjects/keywords'
-
-  feature :disable_holds,
-    default: ENV['DISABLE_HOLDS'],
-    description: 'Determines if holds for local items are currently enabled'
-
-  feature :disable_ills,
-    default: ENV['DISABLE_ILLS'],
-    description: 'Determines if ILLs for local items are currently enabled'
-
-  feature :disable_recalls,
-    default: ENV['DISABLE_RECALLS'],
-    description: 'Determines if recalls for local items are currently enabled'
-
-  feature :disable_scans,
-    default: ENV['DISABLE_SCANS'],
-    description: 'Determines if scans for local items are currently enabled'
-
-  feature :disable_request_digital,
-    default: ENV['DISABLE_REQUEST_DIGITAL'],
-    description: 'Determines if request digital via SFX is currently enabled'
-
-  feature :disable_worldcatinate,
-    default: ENV['DISABLE_WORLDCATINATE'],
-    description: 'Determines if links for renovation items should be enabled'
-
   feature :primo_search,
```

Comment:
I'm curious if both `primo_search` and `primo_redirects` should be removed as features and left as on. I'm happy to hear thoughts on why to keep them though.

Reply:
Resolving this one since we have an open discussion in another thread.